src/utils/multiinvoker.ts

```diff
@@ -294,3 +295,37 @@ export const EmptyInterfaceFee: {
   receiver: Address
   unwrap: boolean
 } = { amount: 0n, receiver: zeroAddress, unwrap: false }
+
+export const mergeMultiInvokerTxs = (
+  transactionData: {
+    data: Hex
+    value: bigint
+    to: Address
+  }[],
+) => {
+  if (transactionData.length === 0) {
```

Comment:
Let's also throw `if (transactionData.some(d => d.to !== transactionData[0].to)`